conda-forge.yml

```diff
@@ -3,12 +3,9 @@ build_platform:
 conda_forge_output_validation: true
 provider:
   win: azure
+  linux_aarch64: azure
 github:
   branch_name: main
   tooling_branch_name: main
 conda_build:
   pkg_format: '2'
-# https://github.com/conda-forge/conda-smithy/pull/1686
```

Comment:
I don't think a new version of conda-smithy has been released. can we keep this please.

Could you also:

```
skip: true  # [no aarch64]
```

and maybe everything except python 3.9 for the next build.